config/cylc_hpc/flow.cylc

```diff
@@ -2,80 +2,76 @@
+            location = bering_strait,beaufort_sea
             startdate = 2022-05-04,2022-05-08
             enddate = 2022-05-09,2022-05-12
             crs = wgs84 #epsg3413 for polar stereographic
             bounding_box=78.2@38.2@70.2@15.2,77.2@37.2@69.2@14.2
             centroid_x = 75,74
             centroid_y = 20,19
-            minfloearea = 300
+            minfloearea = 350
             maxfloearea = 90000
             project_dir = "~/ice-floe-tracker-pipeline"
 
             # Recommend using these default paths for output
+            t = ${CYLC_WORKFLOW_INITIAL_CYCLE_POINT}
             julia_exec = "/usr/local/julia/bin/julia"
             report_dir = $project_dir/"workflow/report"
-            results_dir = $project_dir/"results"
-            fetchdata_dir = $project_dir/"resources"
+            results_dir = $project_dir/"results"/$t
+            fetchdata_dir = $project_dir/"resources"/$t
             truecolor_dir = $fetchdata_dir/"truecolor"
             reflectance_dir = $fetchdata_dir/"reflectance"
-            landmask_dir = $results_dir/"landmasks"
-            preprocess_dir = $results_dir/"preprocess"
-            soit_dir = $results_dir/"soit"
-            tracker_dir = $results_dir/"tracker"
-            h5_dir = $preprocess_dir/"hdf5-files"
-    
+            
     [[global_setup]]  
         script = """
             mkdir -p ~/.cylc
             mkdir -p ~/.cylc/flow
             cp $project_dir/config/cylc_hpc/global.cylc ~/.cylc/flow
         """
     
-    [[mkpaths<job_id>]]
+    [[mkpaths<param_set>]]
         script = """
-            i=${CYLC_TASK_PARAM_job_id}
-            mkdir -p $fetchdata_dir/$i
-            mkdir -p $soit_dir/$i
-            mkdir -p $landmask_dir/$i
-            mkdir -p $preprocess_dir/$i
-            mkdir -p $tracker_dir/$i
-            mkdir -p $h5_dir/$i
+            i=${CYLC_TASK_PARAM_param_set}
+            location_array=( $(echo $location | sed -e 's/,/ /g') )
+            res=$results_dir/${location_array[$i]}
+            mkdir -p $fetchdata_dir/${location_array[$i]}
+            mkdir -p $res/soit
+            mkdir -p $res/landmasks
+            mkdir -p $res/preprocess
+            mkdir -p $res/tracker
+            mkdir -p $res/preprocess/hdf5-files
         """
 
     [[pullfetchimage]]
         script = """
-            apptainer pull --force $project_dir/fetchdata.simg docker://brownccv/icefloetracker-fetchdata:pr-70
+            apptainer pull --force $project_dir/fetchdata.simg docker://brownccv/icefloetracker-fetchdata:main
         """
         platform = oscar
         execution time limit = PT1H
         [[[directives]]]
             --mem = 12G
             --cpus-per-task = 8
 
-    [[fetchdata<job_id>]]
+    [[fetchdata<param_set>]]
         script = """
-            i=${CYLC_TASK_PARAM_job_id}
-
+            i=${CYLC_TASK_PARAM_param_set}
+            location_array=( $(echo $location | sed -e 's/,/ /g') )
```

Comment:
Maybe add a comment describing what this `sed` trick is doing?

Reply:
Great idea, I actually added a comment to the top of each task with a brief description, based on this suggestion!